doc/lang.md

```diff
@@ -495,59 +487,49 @@ module Foo {
 In our example, module `Foo` has four names in its namespace: `N`, `x`,
 `Init`, and `Next`.
 
-A nested module defines its own namespace, which extends the namespace of the
-outer module:
+Quint does not allow for nested modules. However, you can mimick nested
+namespaces by using `::`. For example:
```

Comment:
So if I import a module `Imported`, is it merely mangling the imported names to have an `Imported::` prefix?

Reply:
Conceptually, it should be like that. I don't think it is implemented like this in the lookup table.